runtime/compiler/control/OptionsPostRestore.cpp

```diff
@@ -150,7 +154,19 @@ J9::OptionsPostRestore::iterateOverExternalOptions()
 
          case J9::ExternalOptions::XcompilationThreads:
             {
-            // Need to allocate more if needed
+            char *compThreadsOption = J9::Options::_externalOptionStrings[J9::ExternalOptions::XcompilationThreads];
+            int32_t argIndex = FIND_AND_CONSUME_RESTORE_ARG(EXACT_MEMORY_MATCH, compThreadsOption, 0);
```

Comment:
This doesn't sound right. `EXACT_MEMORY_MATCH` should be used for memory values.

Reply:
I just copied this code from [preProcessCompilationThreads](https://github.com/eclipse-openj9/openj9/blob/9bf20a075d01f0f2377a15c0adad686aa648259c/runtime/compiler/control/J9Options.cpp#L2077-L2078), so it seems we're already ok with `EXACT_MEMORY_MATCH`.